commands/src/api/mod.rs

```diff
@@ -56,4 +56,6 @@ pub enum Body {
     WebRtc(webrtc::Request),
     /// Snapshot commands
     Snapshot(snapshot::Request),
+    /// Gateway deleted command
+    GatewayDeleted,
```

Comment:
Since this is a command, not something like an event or a notification, I think it should use imperative naming such as `DeleteGateway` instead.